python/test/unit/language/test_core.py

```diff
@@ -15,9 +15,22 @@
 
 import triton
 import triton.language as tl
-from triton.runtime.jit import TensorWrapper, reinterpret
 from triton.language.extra import libdevice
 
+from triton._internal_testing import (
+    int_dtypes,
+    uint_dtypes,
+    float_dtypes,
+    dtypes,
+    dtypes_with_bfloat16,
+    torch_float8_dtypes,
+    torch_dtypes,
+    numpy_random,
+    to_triton,
+    torch_dtype_name,
+    to_numpy,
```

Comment:
I refactored some of these helpers so they can be shared with the tma test file, there are no functional changes.